app/Http/Requests/MismatchGetRequest.php

```diff
@@ -4,13 +4,15 @@
 
 use Illuminate\Foundation\Http\FormRequest;
 
-class MismatchesRequest extends FormRequest
+class MismatchGetRequest extends FormRequest
 {
 
     protected function prepareForValidation()
     {
-        $separator = config('mismatches.id_separator');
-        $this->merge(['ids' => explode($separator, $this->ids)]);
+        if ($this->ids) {
```

Comment:
That's right - since the `ids` parameter is even required in GET.

Reply:
Actually, this check really is needed, because otherwise we'd always add the `ids` parameter, even if it isn't there and that way we'll render the `required` validation useless.

Reply:
Then we leave as is and I will mark this as resolved.